pkg/auth/tokenparser.go

```diff
@@ -30,21 +22,7 @@ type TokenClaims struct {
 	OriginalSub       string `json:"original_sub"`
 	UserID            string `json:"user_id"`
 	AccountID         string `json:"account_id"`
-	jwt.StandardClaims
-}
-
-// Valid checks whether the token claims are valid
-func (c *TokenClaims) Valid() error {
```

Comment:
The Valid() function was dropped in the Claim interface in v5. So I move the leeway option to the FromString() function below.